### PR TITLE
added simple Keras / Tensorflow implementation which does the same AE calculations as the torch one

### PR DESCRIPTION
**Pull request by martingrimmer**
- did this to check why the AE has performance problems, does it depend on the used framework?
  - it does seem to produce the same (almost the same) results
- see ids_ae_main.py 
  - used OHE here to exclude Word2Vec factors
  - both implementations generate almost every time bad results for CVE-2017-7529 (19/21)
  - sometimes the results are almost perfect...
    - seems to depend on learning time? or initialisation?
    - we need some experiments on this

### CODE REVIEW

**Review by Darkdragon853 — APPROVED** [review]
Looks fine by me. One left question is: Which datatype has the output of the autoencoder.test_on_batch?

**Comment by martingrimmer** [comment]
`test_on_batch` returns a scalar test loss
